src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/InductionScheduleEntity.kt

```diff
@@ -43,27 +42,22 @@ data class InductionScheduleEntity(
   @Id
   @GeneratedValue
   @UuidGenerator
-  @field:NotNull
```

Comment:
These not null annotations caused me grief in the integration test (for review schedules and completed reviews), but I've removed them from InductionSchedule here as well to prevent similar future issues when we get there